Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -78,14 +76,17 @@ public struct Configuration: Equatable {
         }
         self.disabledRules = validDisabledRules
 
-        self.rules = rules.filter {
-            !validDisabledRules.contains($0.dynamicType.description.identifier)
+        self.rules = rules.filter { rule in
+            let id = rule.dynamicType.description.identifier
+            if validDisabledRules.contains(id) { return false }
+            return enabledRules.contains(id) || !(rule is OptInRule)
```

Comment:
I see. That's not what I was going for, and not what I outlined in https://github.com/realm/SwiftLint/issues/256#issuecomment-163393662.

What you want is a definitive list of rules to run, ignoring `enabled_rules` and `disabled_rules`, which now reading back that comment I linked to, was the motivation for a `rules` config option.

I see value in that, but it does have to be weighed against the complexity of having a white list, a black list, and a list list :disappointed: .